exampleCourse/questions/element/hiddenHints/question.html

```diff
@@ -22,15 +25,28 @@
       Another hint that will be shown immediately.
     </pl-hint>
 
-    <pl-hint show-after-submission=5 hint-name="Final Hint">
-      This is a hint that will be shown after <b>five</b> incorrect submissions for the current question.
-    </pl-hint>
-
     <!-- Comments are allowed inside the hints element -->
 
     <pl-hint show-after-submission=3>
       This is a hint that will be shown after <b>three</b> incorrect submissions for the current question.
     </pl-hint>
+
+    <pl-hint show-after-submission=5 hint-name="Final Hint">
```

Comment:
You shouldn't change the order here, as this is done to demonstrate that the element will reorder hints in the order they are revealed to the student.

Reply:
This hint was originally a few lines before this point and was moved, check the diff.